crates/bevy_sprite/src/picking_backend.rs

```diff
@@ -135,12 +168,44 @@ pub fn sprite_picking(
 
                 let is_cursor_in_sprite = rect.contains(cursor_pos_sprite);
 
-                blocked = is_cursor_in_sprite
+                let cursor_in_valid_pixels_of_sprite = match settings.picking_mode {
+                    SpritePickingMode::AlphaThreshold(cutoff) if is_cursor_in_sprite => {
+                        let texture: &Image = images.get(&sprite.image)?;
+                        // If using a texture atlas, grab the offset of the current sprite index. (0,0) otherwise
+                        let texture_rect = sprite
+                            .texture_atlas
+                            .as_ref()
+                            .and_then(|atlas| {
+                                texture_atlas_layout
+                                    .get(&atlas.layout)
+                                    .map(|f| f.textures[atlas.index])
+                            })
+                            .or(Some(URect::new(0, 0, texture.width(), texture.height())))?;
+                        // get mouse position on texture
+                        let texture_position = (texture_rect.center().as_vec2()
+                            + cursor_pos_sprite.trunc())
+                        .as_uvec2();
+                        // grab pixel
+                        let pixel_index =
```

Comment:
We can replace this with `texture.get_color_at(image_position.x, image_position.y)`. Then we can sample the alpha value through the color. Images can come in all sorts of formats, so we should handle it through `get_color_at`.

Reply:
Valid suggestion. Making the change. (This does unfortunately mean that with .alpha() the config has to be presented as 0-1. More consistent with other areas of Bevy but less so with art programs)